aosc-os-keyring: new, 20241006

MingcongBai opened the pull request:
Topic Description
-----------------

- perl: rebuild for flag change
    Perl records compiler build flags for all future module and extension
    builds at build-time. Rebuild to refresh this, as we have removed
    \`-mfix-loongson3-llsc' for loongson3.
- perl-extutils-cchecker: new, 0.12
- po4a: add perl-syntax-keyword-try dep
- perl-syntax-keyword-try: new, 0.30
- perl-xs-parse-keyword: new, 0.48
- oma: use aosc-os-keyring
- apt: use aosc-os-keyring
    Also remove the old /etc/apt/trusted.gpg.d by checksum matching. It was
    erroneously marked as a conffile.
- aosc-os-keyring: new, 20241006
- ubuntu-keyring: move to runtime-data
- debian-ports-archive-keyring: move to runtime-data

... and 2 more commits

Package(s) Affected
-------------------

- aosc-os-keyring: 20241006
- apt: 2:2.6.1-7
- oma: 1.14.518-1
- perl: 4:5.36.3-4
- perl-extutils-cchecker: 0.12
- perl-syntax-keyword-try: 0.30
- perl-xs-parse-keyword: 0.48
- po4a: 0.73-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit aosc-os-keyring perl perl-extutils-cchecker perl-xs-parse-keyword perl-syntax-keyword-try po4a apt oma
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

MingcongBai committed:
archlinux-keyring: move to runtime-data

MingcongBai committed:
debian-archive-keyring: move to runtime-data

MingcongBai committed:
debian-ports-archive-keyring: move to runtime-data

MingcongBai committed:
ubuntu-keyring: move to runtime-data

MingcongBai committed:
aosc-os-keyring: new, 20241006

MingcongBai committed:
apt: use aosc-os-keyring

Also remove the old /etc/apt/trusted.gpg.d by checksum matching. It was
erroneously marked as a conffile.

MingcongBai committed:
oma: use aosc-os-keyring

MingcongBai committed:
perl-xs-parse-keyword: new, 0.48

MingcongBai committed:
perl-syntax-keyword-try: new, 0.30

MingcongBai committed:
po4a: add perl-syntax-keyword-try dep

MingcongBai committed:
perl-extutils-cchecker: new, 0.12

ruaharico commented:
Works on my amd64 machine
![图片](https://github.com/user-attachments/assets/a4383acc-9f4a-40b7-8b2b-088e491d7567)

MingcongBai commented:
LGTM:

![图片](https://github.com/user-attachments/assets/018c29c5-7459-4e7a-a1c7-3cb9a9db38ad)

MingcongBai commented:
> Works on my amd64 machine ![图片](https://private-user-images.githubusercontent.com/58129639/419123902-a4383acc-9f4a-40b7-8b2b-088e491d7567.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDExMzk5NDAsIm5iZiI6MTc0MTEzOTY0MCwicGF0aCI6Ii81ODEyOTYzOS80MTkxMjM5MDItYTQzODNhY2MtOWY0YS00MGI3LThiMmItMDg4ZTQ5MWQ3NTY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA1VDAxNTQwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQzMTdkNzQxYWZhMTk3YTA1Mjc2NGM4M2YxNGQ2YWU1YTZiNDJhMzZkMjE5YWQ4OTgxM2VkNTcwOGUzYmNhOWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GC1vm4RVBUWeZvCyBFGkyNeoJ6WyualBkjcATufFtOk)

Trouble - APT stops working after `/etc/apt/trusted.gpg.d/aosc.gpg` (a conffile) was recomoved.

MingcongBai committed:
perl: rebuild for flag change

Perl records compiler build flags for all future module and extension
builds at build-time. Rebuild to refresh this, as we have removed
`-mfix-loongson3-llsc' for loongson3.

stdmnpkg reviewed (approved):
works on my x86-64 machine

KexyBiscuit reviewed (approved):
Smoke testing passed on AMD64 with `apt` and `oma`.
